### PR TITLE
feat: server table metadata

### PR DESCRIPTION
**Pull request by AngeloTadeucci**

### CODE REVIEW

File: Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+using System.Collections.Generic;
```
Comment (Zintixx) [inline Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs:1]: nit picky but can we make the directory be named without the _ ?
so ServerTable instead

File: Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+using Maple2.Model.Enum;
+
+namespace Maple2.Model.Metadata;
```
Comment (Zintixx) [inline Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs:4]: namespace would need to be updated to match

File: Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+public record InstanceFieldTable(IReadOnlyDictionary<int, InstanceType> Entries) : ServerTable;
```
Comment (Zintixx) [inline Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs:6]: I think we should be parsing more than just the instance type, no? 
the instanceID may be useful. Not sure but it's definitely in the packet.

File: Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs (hunk 2 above)
Comment (AngeloTadeucci) [reply Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs:4]: can't change namespace, it conflicts with https://github.com/AngeloTadeucci/Maple2/blob/server-table/Maple2.Model/Metadata/ServerTableMetadata.cs#L30

Comment (Zintixx) [reply Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs:4]: ah yeah nvm

**Review by Zintixx — APPROVED** [review]

**Pull request merged**